.github/semantic.yml

```diff
@@ -158,37 +158,20 @@ scopes:
   - vector source # Anything `vector` source related
 
   # transforms
```

Comment:
We'll want to ensure these are also removed from the source: https://github.com/timberio/resources/tree/master/vector/github/semantic_pull_requests.tf

Reply:
Ah yeah, good call. 👍🏻